source/language/types.rst

```diff
@@ -36,10 +36,10 @@ Qubits
 ~~~~~~
 
 There is a quantum bit (``qubit``) type that is interpreted as a reference to a
-two-level subsystem of a quantum state. Quantum registers are static
-arrays of qubits that cannot be dynamically resized. The statement ``qubit name;``
-declares a reference to a quantum bit. The statement ``qreg name[size];`` or ``qubit name[size];`` declares a
-quantum register with the given name identifier. The keyword ``qreg`` is included
+two-level subsystem of a quantum state. The statement ``qubit name;``
+declares a reference to a quantum bit. These qubits are referred to as "virtual qubits" (in distinction to "physical qubits" on actual hardware; see below.) The statement ``qreg name[size];`` or ``qubit name[size];`` declares a
```

Comment:
`[size]` should be attached to `qubit` rather than to the identifier:
```suggestion
declares a reference to a quantum bit. These qubits are referred to as "virtual qubits" (in distinction to "physical qubits" on actual hardware; see below.) The statement ``qreg name[size];`` or ``qubit[size] name;`` declares a
```